docker/kubernetes/README.md

```diff
@@ -0,0 +1,234 @@
+| NAMESPACE |   NAME    | TARGET PORT |            URL            |
+|-----------|-----------|-------------|---------------------------|
+| default   | acme-http | http        | http://192.168.64.3:30160 |
+|-----------|-----------|-------------|---------------------------|
+Opening service default/acme-http in default browser...
+```
+![alt text](images/acme-1-launch.png "Acme gameroom homepage")
+
+In the upper left, click "Register".
+
+Fill out the registration form with values you'll remember. For the
+`Private Key` field, enter the private key information generated for Alice in
+Step 1.
+
+![alt text](images/acme-2-register.png "Acme registration page")
+
+You're logged in as Alice now.
+![alt text](images/acme-3-loggedin.png "Alice logged in page")
+
+Repeast steps for Bubba/Bob.
+
+`$ minikube service bubba-http`
+
+![alt text](images/bubba-1-launch.png "Bubba gameroom homepage")
+
+![alt text](images/bubba-2-register.png "Bubba registration page")
+
+![alt text](images/bubba-3-loggedin.png "Bob logged in page")
+
+### Step 6: Create a gameroom
+
+Switch back to the Acme instance. Click the `+` next to `My Gamerooms`.
+Select Bubba from the dropdown menu and give your new gameroom a name.
+Then click Send.
+
+![alt text](images/acme-4-newgameroom.png "Creating a new gameroom")
+
+After clicking Send, you'll see a green notification indicating that the
+invitation has been successfully sent.
+
+![alt text](images/acme-5-invitationsent.png "Invitation successfully sent")
+
+Click `Invitations` in the bottom left, then `SENT`. The invitation is visible
+in the list.
+
+![alt text](images/acme-6-invitationlist.png "Alice's sent invitation list")
+
+Switch to the Bubba instance. You'll see that the bell in the upper right has an
+activity indicator. Click this to view the notification, then click the
+notification to view the invitation.
+
+![alt text](images/bubba-4-invitationnotification.png "Bob's invitation notification")
+
+Click the "Accept" to accept the invitation. After a brief notification the
+invitation will disappear.
+
+![alt text](images/bubba-5-invitation.png "Bob's invitation")
+
+Click the newly created gameroom in the left sidebar to view the gameroom.
+
+![alt text](images/bubba-6-gameroom.png "Bob's view of the new gameroom")
+
+### Step 7: Play Tic-tac-toe
+
+Switch back to the Acme instance. Click the new gameroom in the left sidebar.
+Then click the "New Game" button to start a new game of tic-tac-toe. Enter a
+name for your game and click "Send"
+
+![alt text](images/acme-7-creategame.png "Alice creates a new game")
+
+Click anywhere on the game to join and then select a square to begin.
+
+![alt text](images/acme-8-joingame.png "Alice joins the new game")
+
+![alt text](images/acme-9-takespace.png "Alice takes a space")
+
+Switch to the Bubba instance to continue.
+
+![alt text](images/bubba-7-joingame.png "Bob joins the new game")
+
+![alt text](images/bubba-8-takespace.png "Bob takes a space")
```

Comment:
A summary sentence would help finish this excellent doc in style. (Perhaps something like "You can continue playing this game, start new games, and or quit in a huff.") Better yet, how about explaining how to shut down the demo?